environment2.yml

```diff
@@ -26,6 +26,7 @@ dependencies:
 - spyder
 - numexpr
 - pip:
+  - dipy
```

Comment:
please also add `dipy` to the `doc/conf.py` intersphinx list. Check that it works properly by doing in the `doc` folder` make html_dev-noplot`. You should see it successfully retrieve the `objects.inv` list.

Reply:
If I add to `conf.py` in the `intersphinx_mapping`:
```
    'dipy': ('https://nipy.org/dipy', None),
```
I don't get any warnings with `make html_dev-noplot` other than the standard 5 about images missing (because not all examples were built; plus one about `massich` that is fixed in `master`), and `dipy` links are in the rendered example file. What warnings do you see?

Reply:
Sorry URL is `http` not `https`

Reply:
a lot of those:
path/mne-python/doc/auto_examples/time_frequency/plot_compute_csd.rst:17: WARNING: py:func reference target not found: mne.time_frequency.csd_fourier 

and those:
path/mne-python/doc/documentation.rst:: WARNING: toctree contains reference to nonexisting document u'generated/csd_morlet'

Reply:
I added exactly what you've added... but I guess that's not the problem anyway...

Reply:
What versions of `sphinx` and `numpydoc` are you using?

Reply:
sphinx: 1.7.2
numpydoc: 0.7.0

Reply:
oh yes, and I had already http (just claimed it to be "exactly" the same, but this was the only difference ;) )

Reply:
0.7 might be too old, can you try upgrading to see if it helps?

Reply:
0.8 seems the newest but yields the same warnings...

Reply:
Okay there must be something wrong with your config we can fix later. In the meantime you can make these changes and CircleCI should built it properly